src/app/components/homepage/carousel/carousel.component.css

```diff
@@ -0,0 +1,14 @@
+img{
```

Comment:
bad practice to define styles using element selectors - use classes instead - as this will cause problems when you need to add additional buttons or anchor links to your header